### PR TITLE
Add initial implementation

### LINKED ISSUE #2: Add initial implementation

### PR DESCRIPTION
**Pull request by lhorie**
Resolves #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+  if (ctx.element) {
+    const userData = await getUserData();
+    // userData can't be trusted, and is automatically escaped
+    ctx.body.body.push(html`<div>${userData}</div>`)
+  }
+  return next();
+}
+```
+
+If `userData` above was `<script>alert(1)</script>`, the string would be automatically turned into `<div>&lt;script&gt;alert(1)&lt;/script&gt;</div>`. Note that only `userData` is escaped, but the HTML in your code stays intact.
```
Comment (rtsao) [inline README.md]: The escape method is different now

File: README.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+Note that you cannot mix sanitized HTML with unsanitized strings:
+
+```js
+ctx.body.body.push(html`<h1>Safe</h1>` + 'not safe') // will throw an error when rendered
+```
+
+Also note that only template strings can have template tags (i.e. <code>html&#x60;&lt;div&gt;&lt;/div&gt;&#x60;</code>). The following are NOT valid Javascript: `html"<div></div>"` and `html'<div></div>'`.
+
+If you get an <code>Unsanitized html. You must use html&#x60;[your html here]&#x60;</code> error, remember to prepend the `html` template tag to your template string.
```
Comment (rtsao) [inline README.md]: This also I think is not in sync

File: README.md (hunk 1 above)
Comment (lhorie) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (lhorie) [reply README.md]: No, this is fine. It's a quirk of markdown syntax that we need to html-escape backticks and angled brackets when we want to actually them print out

Comment (rtsao) [reply README.md]: Ah, got it 👍

File: test/fixtures/syntax-error/.fusion/stats.json [hunk 3]
```diff
@@ -0,0 +1,6519 @@
...
.../node_modules]\n[/Users/lhorie/Documents/graphene/packages/core/test/node_modules]\n[/Users/lhorie/Documents/graphene/packages/node_modules]\n[/Users/lhorie/Documents/node_modules]\n[/Users/lhorie/node_modules]\n[/Users/node_modules]\n[/node_modules]\n[/Users/lhorie/Documents/graphene/packages/core/node_modules/fusion-cli/entries/client-entry]\n[/Users/lhorie/Documents/graphene/node_modules/fusion-cli/entries/client-entry]\n[/Users/lhorie/Documents/graphene/packages/core/node_modules/fusion-cli/entries/client-entry.js]\n[/Users/lhorie/Documents/graphene/node_modules/fusion-cli/entries/client-entry.js]\n[/Users/lhorie/Documents/graphene/packages/core/node_modules/fusion-cli/entries/client-entry.json]\n[/Users/lhorie/Documents/graphene/node_modules/fusion-cli/entries/client-entry.json]\n[/Users/lhorie/Documents/graphene/packages/core/node_modules/fusion-cli/entries/client-entry]\n[/Users/lhorie/Documents/graphene/node_modules/fusion-cli/entries/client-entry]\n @ ./src/test/browser/index.js 2:10-52\n @ ./src/test/browser ^\\.\\/.*$\n @ /Users/lhorie/Documents/graphene/packages/core/entries/client-test.js\n @ multi /Users/lhorie/Documents/graphene/packages/core/entries/client-test.js",
```
Comment (rtsao) [inline test/fixtures/syntax-error/.fusion/stats.json]: Should these files be here?

**Comment by lhorie** [comment]
!merge

**Pull request merged**